backend/src/utils/tokens.ts

```
@@ -0,0 +1,80 @@
+ * Generate JWT access token
+ */
+export const generateAccessToken = (payload: any): string => {
+  return jwt.sign(payload, config.auth.jwtSecret, {
+    expiresIn: config.auth.jwtExpiresIn,
+  } as any);
+};
+
+/**
+ * Verify JWT token
+ */
+export const verifyToken = (token: string): any => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)